src/cluster-configuration/deploy/priority-class.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: pai-daemon-service-priority
+value: 1000000
+globalDefault: false
+description: "This priority class should be used for PAI daemon service only."
```

Comment:
could you use the built in priority `system-node-critical`?

Reply:
This priority is limited to `kube-system` namespace. We can use `system-node-critical`priority in other namespace after 1.1.7 version.

Refer to
https://github.com/kubernetes/kubernetes/issues/60596#issuecomment-559537077